.travis.yml

```diff
@@ -53,7 +53,7 @@ script:
   - npm test
 
 after_success:
-  - if [ $TRAVIS_OS_NAME == "linux" ] && [ $NODE_VERSION == "4" ]; then
+  - if [ $TRAVIS_OS_NAME == "linux" ] && [ $NODE_VERSION == "6" ]; then
```

Comment:
Is there is reason we run the script against node 6 ? Or can we remove the node version check entirely ?

Reply:
Good question I will check

Reply:
It is flagged so that we don't generate a coverage report for every version of node.  It would generate unnecessary files and take time.

@junedkazi unless you object I am going to leave it in place